Author: Andreas Tille <tille@debian.org>
Last-Update: Fri, 06 Nov 2020 09:54:26 +0100
Description: Leave out minor bazel version
 Otherwise bazel commands all fail with 'version not found'

--- a/.bazelversion
+++ b/.bazelversion
@@ -1,2 +1,2 @@
-6.1.0
-# NOTE: Update Bazel version in tensorflow/tools/ci_build/release/common.sh.oss
\ No newline at end of file
+4
+# NOTE: Update Bazel version in tensorflow/tools/ci_build/release/common.sh.oss
